### PR TITLE
Update to 0.5.0 release + new logo + new spelling

### PR DESCRIPTION
**Pull request by Nirusu**
marblerun.sh is quite outdated right now... So here's a refresh that should update some of the things we updated on our other product pages.

### CODE REVIEW

**Review by m1ghtym0 — APPROVED** [review]

File: content/_index.md [hunk 1]
```diff
@@ -11,8 +11,8 @@ buttons:
     caption: Join the community
     url: '/community/'
 news:
-  title: "Announcing [Marblerun 0.3.2](/docs/changelog/)"
-  content: "Adds support for [Occlum](https://www.occlum.io)! Check out the [blog post](https://edgelesssys.medium.com/marblerun-now-supports-occlum-even-more-confidential-computing-at-scale-2f6dd17e00c0)."
+  title: "Announcing [MarbleRun 0.5.0](https://github.com/edgelesssys/marblerun/releases/tag/v0.5.0)"
+  content: "Now featuring support for Alibaba Cloud!"
```
Comment (thomasten) [inline content/_index.md:15]: Does this really work @daniel-weisse ?
Is this the most exciting feature?

Comment (daniel-weisse) [reply content/_index.md:15]: As long as Intel doesnt fix their PCCS implementation we cant verify quotes, only generate them.
To be perfectly honest the support was there before for standalone installations, it's only the docker file that changed, as well as better integration for different Kubernetes device plugins.

Maybe change it from specifically Alibaba Cloud support to generalized Intel DCAP support? (doesn't sound quite as nice though)

Comment (Nirusu) [reply content/_index.md:15]: Heh, this was the best I could come up with on the go :)

Generalized Intel DCAP support does not quite sound as nice... maybe the revised file provisioning? Was thinking about mentioning this, but did not know how to phrase it nicely.

**Review by flxflx — APPROVED** [review]

**Comment by Nirusu** [comment]
Since this is mostly ready for release, which tagline for the update do we go with? Which feature do we want to highlight?

**Pull request merged**